indico/modules/events/templates/reviews/_omnibox.html

```diff
@@ -72,7 +72,7 @@
                         {% if group in reviewer_data.reviewed_groups %}
                             {% set review = reviewer_data.reviews[group] %}
                             <a class="js-new-edit-review" data-review-id="{{ review.id }}">
-                                {{ group.full_title }}
+                                {{ group.full_title if group.full_title else group.title }}
```

Comment:
Not needed anymore. `group.full_title` falls back to `group.title` at the proxy level.